src/main/java/uk/gov/hmcts/reform/civil/handler/callback/user/RespondToClaimSpecCallbackHandler.java

```diff
@@ -430,6 +440,7 @@ private CallbackResponse setApplicantResponseDeadline(CallbackParams callbackPar
 
         return AboutToStartOrSubmitCallbackResponse.builder()
             .data(updatedData.build().toMap(objectMapper))
+            .state("AWAITING_APPLICANT_INTENTION")
```

Comment:
Is there an ENUM for states?

Reply:
You coud use AWAITING_APPLICANT_INTENTION.name() which would point to the enum